packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -7,23 +7,32 @@
 #import <AVFoundation/AVFoundation.h>
 #import <MobileCoreServices/MobileCoreServices.h>
 #import <Photos/Photos.h>
+#import <PhotosUI/PHPhotoLibrary+PhotosUISupport.h>
+#import <PhotosUI/PhotosUI.h>
 #import <UIKit/UIKit.h>
 
 #import "FLTImagePickerImageUtil.h"
 #import "FLTImagePickerMetaDataUtil.h"
 #import "FLTImagePickerPhotoAssetUtil.h"
 
-@interface FLTImagePickerPlugin () <UINavigationControllerDelegate, UIImagePickerControllerDelegate>
+@interface FLTImagePickerPlugin () <UINavigationControllerDelegate,
+                                    UIImagePickerControllerDelegate,
+                                    PHPickerViewControllerDelegate>
 
 @property(copy, nonatomic) FlutterResult result;
 
+@property NSDictionary *_arguments;
+
+@property(readonly) PHPickerViewController *_pickerViewController;
+
+@property(readonly) BOOL _phPickerFlag;
```

Comment:
I don't think we need this as a property or ivar.
We can be a local variable and pass along the methods.

Reply:
Done!